packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/WorkingGroupLeadOpening/types.ts

```diff
@@ -11,3 +11,7 @@ export interface WorkingGroupAndOpeningDetailsParameters {
   shortDescription?: string
   groupId?: string
 }
+
+export interface CancelWorkingGroupLeadOpeningParameters extends WorkingGroupAndOpeningDetailsParameters {
+  openingId?: string
+}
```

Comment:
This should be a new type. We only need `groupId` and `openingId` for the transaction. The descriptions are for the opening details and not needed for canceling and opening.